src/05_lists.py

```diff
@@ -8,22 +8,28 @@
 
 # Change x so that it is [1, 2, 3, 4]
 # YOUR CODE HERE
+x.append(4)
 print(x)
 
 # Using y, change x so that it is [1, 2, 3, 4, 8, 9, 10]
 # YOUR CODE HERE
+x.extend(y)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 10]
 # YOUR CODE HERE
+x.pop(4)
 print(x)
 
 # Change x so that it is [1, 2, 3, 4, 9, 99, 10]
 # YOUR CODE HERE
+x.insert(4, 99)
```

Comment:
Looks like this gets you `[1, 2, 3, 4, 99, 9, 10]` as an output; super small difference, but if you have a target output try to be sure that's what you're getting